style.css

```diff
@@ -8,25 +8,54 @@
   scroll-behavior: smooth;
 }
 
+a {
+  cursor: pointer;
+}
+
 body {
-  width: 375px;
+  display: flex;
+  justify-content: center;
   background-color: #e5e5e5;
 }
 
+.status-bar {
+  display: block;
+}
+
+@media screen and (min-width: 768px) {
```

Comment:
- [x] [OPTIONAL] I noticed multiple usage of the @media screen and (min-width: 768px) in your styling. I would strongly suggest you use only one `@media screen and (min-width: 768px)` and have all your styling properties nested inside, with this, you will follow the `DRY` method of the best HTML and CSS coding practices